independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -662,12 +713,19 @@ public void close() {
                 log.error("Failed to close " + element, e);
             }
         }
-        ResourceBundle.clearCache(this);
-
+        classPathElements.clear();
```

Comment:
if we are doing that we could as well simply de-reference those instead of clearing

Reply:
Setting them to null would probably be problematic when we access the CL after it has been closed.

I'm not exactly optimistic on our capability to fix all the problematic cases soonish (if ever).

Reply:
you could use `List.of()` but i am not sure an empty list will actually work, we'll see what the CI says